lib/wallaby/phantom.ex

```diff
@@ -1,13 +1,45 @@
 defmodule Wallaby.Phantom do
+  @moduledoc """
+  Wallaby driver for PhantomJS.
+
+  ## Usage
+
+  Start a Wallaby Session using this driver with the following command:
+
+  ```
+  {:ok, session} = Wallaby.start_session(driver: Wallaby.Phantom)
+  ```
+
+  ## Notes
+
+  This driver requires PhantomJS be installed in your path. You can install PhantomJS through NPM or your package manager of choice:
```

Comment:
@PragTob I'm not sure I understand what these docs should look like. I copied these notes from the readme and have an example of setting the `:phantomjs` config value on line 24. What would be a good way to improve the clarity here?